### PR TITLE
lower deployment target to 8.0, the oldest supported by XCode 8

### PR DESCRIPTION
**Pull request by mman**
This pull request makes it possible to use Typist in apps that need to support iOS 8.

### CODE REVIEW

File: Typist.swift [hunk 1]
```diff
@@ -119,8 +119,10 @@ public class Typist: NSObject {
     
     internal func keyboardOptions(fromNotificationDictionary userInfo: [AnyHashable : Any]?) -> KeyboardOptions {
         var currentApp = false
```
Comment (totocaster) [inline Typist.swift:121]: I think this line should be changed to `true` since keyboard always belong to current app in iOS 8.

**Review by totocaster — CHANGES_REQUESTED** [review]
Hey Martin, thanks for PR! Before I merge, can you please make so `belongsToCurrentApp` reflects reality in iOS 8? (See line comment in review.) Thanks again for contribution!

**Comment by mman** [comment]
Very good catch, thanks,

I've updated the pull request to correctly report that we are the current app showing the keyboard when running under iOS < iOS 9...

**Comment by totocaster** [comment]
Great! Thank you! Will include in upcoming release.

**Comment by mman** [comment]
Excellent, thanks :-)

**Pull request merged**